history/provinces/00_k_gilneas.txt

```diff
@@ -63,7 +63,11 @@
 	culture = gilnean
 	religion = holy_light
 	holding = castle_holding
+	special_building_slot = greymane_wall_01
 
+	600.1.1 = {
+		special_building = greymane_wall_02
```

Comment:
The wall is under Arugal's control

Reply:
Should I move it to Crowley then?
Its location is quite confusing